engine/table/src/test/java/io/deephaven/engine/util/TestTypeUtils.java

```diff
@@ -134,15 +134,15 @@ public void testIsType() {
         assertTrue(io.deephaven.util.type.TypeUtils.isPrimitiveNumeric(int.class));
         assertFalse(io.deephaven.util.type.TypeUtils.isPrimitiveNumeric(Double.class));
 
-        assertFalse(io.deephaven.util.type.TypeUtils.isBoxedNumeric(Instant.class));
-        assertFalse(io.deephaven.util.type.TypeUtils.isBoxedNumeric(Date.class));
-        assertFalse(io.deephaven.util.type.TypeUtils.isBoxedNumeric(int.class));
-        assertTrue(io.deephaven.util.type.TypeUtils.isBoxedNumeric(Double.class));
+        assertFalse(NumericTypeUtils.isBoxedNumeric(Instant.class));
```

Comment:
Should we be pedantic and move these to a new class?

Reply:
[:duck:](https://www.simplethread.com/looks-great-lose-the-duck/)